app/controllers/application_records_controller.rb

```diff
@@ -132,4 +130,13 @@ def given_or_all_department_ids
   def record_params
     params.permit(:position_id, :ethnicity, :gender)
   end
+
+  def modify_params
```

Comment:
This could use a more specific name. `modify_params` doesn't offer too much information about what it does or to what controller method(s) it's applicable. I can offer some suggestions if you need.